src/views/product-integration/component-view/components/variable-group-list/style.module.css

```diff
@@ -90,3 +81,19 @@
 		}
 	}
 }
+
+.key {
+	composes: g-offset-scroll-margin-top from global;
+	border: 1px solid var(--token-color-border-primary);
+	border-radius: 5px;
+	padding: 2px 4px 3px;
+	background: var(--token-color-palette-neutral-50);
+	font-size: 16px;
+	color: var(--token-color-foreground-strong);
+}
+
+.permalink {
+	@nest .topRow:hover & {
```

Comment:
When anything in the `.topRow` is hovered, the `.permalink` element within that `.topRow` will be shown.